PerformanceTest_Static_PressureDorsal.py

```diff
@@ -130,36 +140,36 @@ def createAvgMat(inputName):
     return(result)
 
 
-# meanDorsalPressure = []
-# maxDorsalPressure = [] 
-# sdDorsalPressure = []
-# totalDorsalPressure = []
-# config = []
-# subject = []
-# Movement = []
+meanDorsalPressure = []
+maxDorsalPressure = [] 
+sdDorsalPressure = []
+totalDorsalPressure = []
+config = []
+subject = []
+Movement = []
 
-# plantarContact = []
-# plantarPeakPressure = []
-# plantarAvgPressure = []
-# plantarSDPressure = []
-# plantarTotalPressure = []
+plantarContact = []
+plantarPeakPressure = []
+plantarAvgPressure = []
+plantarSDPressure = []
+plantarTotalPressure = []
 
 heelArea = [] 
 heelAreaUP = []
 
-for entry in entries:
+for entry in entries[1:]:
```

Comment:
we can add #entries.pop(0) within the for loop to avoid having to switch between entries[1:] and entries: